CHANGELOG.md

```diff
@@ -8,7 +8,8 @@
 
 - Fixed a bug in `session.read.csv` that caused an error when setting `PARSE_HEADER = True` in an externally defined file format.
 - Fixed a bug in query generation from set operations that allowed generation of duplicate queries when children have common subqueries.
-
+- Fixed a bug in `pyspark.sql.functions.array_remove` to remove is available in SQL but there is no corresponding API.
+- 
```

Comment:
we don't have `pyspark.sql.functions`. Also, this is not a bug. Please remove this log line.

Reply:
I added "New Features"  and moved the new the feature to there.